nevergrad/optimization/differentialevolution.py

```diff
@@ -257,6 +257,7 @@ def __init__(
 
 
 DE = DifferentialEvolution().set_name("DE", register=True)
+DDE = DifferentialEvolution(crossover="dimension").with_name("DDE", register=True)
```

Comment:
Can you tell me a bit more about this line ?

Reply:
Ah this is "set_name", not "with_name".

Reply:
I'll remove this, it has nothing to do in this PR sorry